src/main/java/com/blog/web/models/UserEntity.java

```diff
@@ -0,0 +1,32 @@
+package com.blog.web.models;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity(name = "users")
+// Named UserEntity to prevent conflicts with Java User object
+public class UserEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
add final to all constructors if you don't need the setter -> ensures immutability

Reply:
marked as resolved but adjustment wasnt done